timemachine/lib/__init__.py

```diff
@@ -20,6 +20,18 @@ def impl(self):
         return custom_ops.LangevinIntegrator(self.dt, self.ca, self.cbs, self.ccs, self.seed)
 
 
+class VelocityVerletIntegrator:
+    def __init__(self, dt, masses):
+        self.dt = dt
+
+        cb = dt / masses
+        cb *= -1
+        self.cbs = cb
+
+    def impl(self):
+        return custom_ops.VelocityVerletIntegrator(self.dt, self.cbs)
```

Comment:
Not a huge fan of the `self.cb`, `self.cbs` naming convention / sign conventions here and in the other implementations -- seems like this is exposing an internal implementation detail that could be kept internal.

Would it still be convenient if these both had the same constructor arguments: `custom_ops.VelocityVerletIntegrator(dt, masses)`, rather than `custom_ops.VelocityVerletIntegrator(dt, -dt/masses)`?

Reply:
I was trying to stay consistent with the Langevin integrator. We could move the pre-processing of the reused parameters to the C++. And would you suggest something else for the name, I certainly don't find it very obvious what it is?

Reply:
>We could move the pre-processing of the reused parameters to the C++.

Non-blocking of course, but that sounds preferable to me (perhaps for a later PR, since it might involve either introducing an inconsistency with Langevin or making more changes to Langevin).

(I also may be missing if there's a reason it's more convenient to do these computations in the Python layer.)

>And would you suggest something else for the name, I certainly don't find it very obvious what it is?

No preference. I hadn't ever thought to name `dt / m` (or `-dt / m`), and I assumed the names `ca`, `cb`, `cc` from `langevin_coefficients` were arbitrary.

Reply:
Created issue https://github.com/proteneer/timemachine/issues/792